requirements.in

```diff
@@ -1,7 +1,7 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements.in
 
-Flask==1.0.4
+Flask>=1.1,<2
 Flask-Login==0.5.0
 Flask-WTF==0.14.3
 itsdangerous==1.1.0
```

Comment:
Can we remove this pin as well? I think Flask now controls the version of `itsdangerous` as of 1.1.4: https://github.com/pallets/flask/blob/1.1.4/setup.py#L58

Otherwise looks sensible.

Reply:
Ah, that's what changed. Will remove now